docs/sources/mimir/references/http-api/index.md

```diff
@@ -1223,6 +1225,22 @@ The `blocks_deleted` field will be set to `true` if all the tenant's blocks have
 
 Requires [authentication](#authentication).
 
+### Compactor tenants
+
+```
+GET /compactor/tenants
+```
+
+Displays a web page with the list of tenants with blocks in the storage configured for compactor.
+
+### Compactor tenant planned jobs
+
+```
+GET /compactor/tenant/{tenant}/planned_jobs
+```
+
+Displays a web page listing planned compaction jobs computed from current bucket-index for a given tenant.
```

Comment:
I think _bucket index_ is two words: https://grafana.com/docs/mimir/latest/references/architecture/bucket-index/

```suggestion
Displays a web page listing planned compaction jobs computed from the bucket index for the given tenant.
```